good-enough-practices-for-scientific-computing.tex

```diff
@@ -127,47 +127,43 @@ \subsection*{Acknowledgments}
 \section{Data Management}\label{sec:data}
 
 Data within a project may need to exist in various forms, ranging from what
-first arrives to what is included in the final publication.  The aim of
-our recommendations is to reproducibly create ``tidy data'', which can be a
-powerful accelerator for analysis \cite{wickham2014,hart2015}.
+first arrives to what is actually used for the primary analyses.  Our recommendations have
+two main themes. One is to work towards ready-to-analyze data incrementally,
+documenting both the intermediate data and the process. We also describe the
+key features of ``tidy data'', which can be a powerful accelerator for analysis
+\cite{wickham2014,hart2015}.
 
 \begin{enumerate}
 
 \item
-  \recommend{Save data in the raw form}.  Where possible,
+  \recommend{Save the raw data}.  Where possible,
   save data as originally generated (i.e. by an instrument or from a survey).
   It is
   tempting to overwrite raw data files with cleaned-up versions, but
-  faithful retention is essential to being able to re-run analyses
-  from start to finish; to recover from analytical
-  mishaps; and for experimenting without fear. Consider restricting file permissions to read-only,
-  even (or especially) for yourself, so it is harder to damage raw
+  faithful retention is essential for re-running analyses
+  from start to finish; for recovery from analytical
+  mishaps; and for experimenting without fear. Consider changing file permissions to
+  read-only or using spreadsheet protection features,
+  so it is harder to damage raw
   data by accident or to hand edit it in a moment of weakness.
 
-  If the data is large and in a format that is inefficient to store or process,
-  transform it for storage with a lossless, well-documented procedure---avoid
+  Some data will be impractical to manage in this way. For example, you should avoid
```

Comment:
Got rid of recommendation to "transform with a lossless, well-documented procedure" because I had no idea what we meant. If someone wants to bring that back, we need a specific example.